sys/event/Makefile

```diff
@@ -0,0 +1,5 @@
+SRC := event.c
+
+SUBMODULES = 1
```

Comment:
I don't think that this is managed pretty transparent here. For `core` I was understanding of that, but in `sys` the sub-folders for sub-modules is already established (also you don't need pseudo-modules then ;-)).

Reply:
I don't understand transparent.
Using submodules here saves two folders and two makefiles with 2 lines each.

Reply:
But makes it harder for newcomers to understand what to include ... (that's what I meant with transparent)